src/legacy/core_plugins/data/public/search/search_bar/lib/saved_query_service.ts

```diff
@@ -58,6 +58,11 @@ export const createSavedQueryService = (
       query,
     };
 
+    if (!attributes.title.length) {
```

Comment:
Let's move this condition to the very beginning of the function. It's best to fail as early as possible.

Reply:
https://github.com/elastic/kibana/pull/45029/commits/af7243288d5f53acd58e65c56cb8a4ca05e68c40